### ISSUE TITLE
No profile pictures anywhere else but the scoreboard

### ISSUE THREAD

**Issue by oWind-RL** [mmiddletonn/RLProfilePictures#2]
Hey! ive been using this plugin for a few days and ive noticed that it dosent display the profile pictures anywhere else but the scoreboard. is this on purpose? or is this a problem?

**Comment by squidnugget07** [comment]
well thats where your most likely going to see the profile picture, middleton prob cant do it because of avatar borders

**Comment by mmiddletonn** [comment]
Yeah, I could add overlays in other places but it would be messy. I am working on another version of the plugin that goes around bakkesmod's sdk to modify the avatar textures directly. It would be a much cleaner result and would make them appear in more places, but since it uses external sdks, bakkesmod likely won't allow it on their site. I will have more information soon.
